src/bgw_policy/policy.c

```diff
@@ -6,8 +6,10 @@
 
 #include <postgres.h>
 #include <utils/builtins.h>
+#include <jsonb_utils.h>
 
 #include "bgw/job.h"
+#include "dimension.h"
```

Comment:
Is those headers necessary? Asking because you're just including and didn't touch other parts of this file.

Reply:
You're right, they're not. Removed.